src/js/_store/modules/articles.js

```diff
@@ -51,6 +52,9 @@ export default {
     deleteArticleId: state => state.deleteArticleId,
   },
   mutations: {
+    getTrashedArticles(state, payload) {
+      state.trashedArray.push(...payload);
```

Comment:
[nits]
一度展開しなくても、
```suggestion
      state.trashedArray = payload;
```

で良いかなと思いました！

Reply:
修正いたしました！
fe3c8dc0769fd06b6fe6e9d94ca3f2f1990b5d22